### ISSUE TITLE
Keyboard not displayed on mobile device

### ISSUE THREAD

**Issue by ZiggyQubert** [germanoid/germanoid.github.io#4]
When running this on a mobile device the keyboard wont display and it cant be used :(

By far the best version of this I have seen and would be great to be able to run this on your phone while playing the game.

**Comment by germanoid** [comment]
Yes, for now, you need a physical keyboard (Bluetooth, OTG) or a keyboard you can open any time. [Hacker's Keyboard](https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard&hl=en) had an option for that.

The problem is, that Android only open the Keyboard when you have an input field. In the first proof of concept, I was using an input box, but it felt extremely clunky. The android autocompletion where a huge problem too. So until I found a way tricking the smartphone into open the keyboard we have to live with what we have. I love and hate smartphones at the same time : )
